newsletter-april-2021.rst

```diff
@@ -0,0 +1,116 @@
+   and spread the word about our service.
+-  The average daily pageviews went up by 9.4 % from the previous month.
+-  We have `a new status page <http://status.readthedocs.com/>`_ so you
+   can see the availability history and current status of our infrastructure.
+
+New features
+------------
+
+-  There is a `new support form <https://docs.readthedocs.io/en/latest/support.html>`_
+   in case you need to send us a request related to your project or
+   account, like asking for more resources, change the project slug, and
+   so forth.
+-  You can now build your project using Python 3.9 by setting your build
+   image to ``testing``. Please `open an
+   issue <https://github.com/readthedocs/readthedocs.org/issues/new>`_
+   if you find any problems with it.
+-  Now `all newly created MkDocs projects use the latest version of mkdocs by
+   default <https://github.com/readthedocs/readthedocs.org/pull/7869>`_.
+   Existing projects can specify a specific version, but won't automatically
+   be upgraded.
+-  We expanded a few areas of our user documentation: we added `a brand
+   new guide on reproducible
+   builds <https://docs.readthedocs.io/en/stable/guides/reproducible-builds.html>`_,
+   considerably improved our `guide on using
+   conda <https://docs.readthedocs.io/en/stable/guides/conda.html>`_,
+   and `started recommending MyST for writing Markdown with
+   Sphinx <https://docs.readthedocs.io/en/stable/intro/getting-started-with-sphinx.html#using-markdown-with-sphinx>`_.
+-  We added the possibility to declare `publicly visible environment
+   variables <https://github.com/readthedocs/readthedocs.org/pull/7891>`_.
+
+You can always see the latest changes to our platforms in our `Read the Docs
+Changelog <https://docs.readthedocs.io/page/changelog.html>`_ and `Ethical Ad Server
+Changelog <https://ethical-ad-server.readthedocs.io/page/developer/changelog.html>`_.
+
+Current focus & known issues
+----------------------------
+
+-  We have had some availability issues with our commercial hosting for the past
+   few weeks. This was caused by massive I/O load spikes on our web servers,
+   causing increased CPU contention and slower web requests. We tracked the issue
+   down to a background process that was doing a lot of reads from our cloud
+   storage. We have changed our infrastructure so background tasks and web
+   requests are processed by different servers, which has addressed the stability
+   issues.
+-  We were made aware of an open redirect on documentation pages via
+   a report to our ``security`` address. This was present in our codebase for the
+   last 2 releases, and was fixed in a deployment on Thursday April 1. We'd like
+   to thank Splunk and the Cryptography project for reporting this issue through
+   proper channels. Please always follow our `security reporting guide
+   <https://docs.readthedocs.io/en/latest/security.html>`_ to report potential
+   security issues.
+
+Upcoming features
```

Comment:
I was thinking if there is a better/more generic word instead of "features", since there are some in this list that I think are not features. Maybe "Upcoming work"?

Reply:
Roadmap? Focus?

Reply:
Yeah, I was thinking about this as well but couldn't come up with a better name. "Roadmap" sounds a bit more long term to me, and I'm not in love with "Focus" either. "Coming soon" sounds a bit too informal I guess. "Next steps"? "Current priorities"?

Reply:
I think this is fine for now, and we can adapt in the future as we figure it out.